DigitalLearningSolutions.Web/Views/Shared/_MyAccountVerifyCentreEmails.cshtml

```diff
@@ -0,0 +1,34 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common.ViewComponents
+@model VerifyEmailWarningIfAppropriateViewModel
+
+@{
+  var numSpecificPhrases = new {
+    address = Model.MultipleEmailsAreUnverified ? "addresses" : "address",
+    thisAddress = Model.MultipleEmailsAreUnverified ? "each of the addresses above" : "this address",
+    link = Model.MultipleEmailsAreUnverified ? "links" : "link",
+    folder = Model.MultipleEmailsAreUnverified ? "folders" : "folder",
+    it = Model.MultipleEmailsAreUnverified ? "them" : "it",
+    email = Model.MultipleEmailsAreUnverified ? "emails" : "email",
+  };
+}
+
+@if (Model.AtLeastOneCentreEmailIsUnverified) {
+  foreach (var ((centreName, email), index) in Model.UnverifiedCentreEmails.Select((email, index) => (email, index))) {
+    <p>
+      Your email address @(email) has
+      @(Model.PrimaryEmailIsUnverified && index >= 0 || !Model.PrimaryEmailIsUnverified && index >= 1 ? "also" : "")
+      not been verified yet.
+      You will not be able to access your account at @(centreName) until you verify the address.

```

Comment:
_Probably_ makes more sense to users to say, "You will not be able to access @(centreName) until..."

Also, can we make the centre name bold? Or does that look weird?

Reply:
done. I like the bold centre name
![localhost_44363_MyAccount](https://user-images.githubusercontent.com/70278044/182105691-114e784b-87fd-4479-9fd9-8f3e5909f019.png)